apple/internal/rule_factory.bzl

```diff
@@ -40,6 +40,12 @@ load(
 # Returns the common set of rule attributes to support Apple test rules.
 # TODO(b/246990309): Move _COMMON_TEST_ATTRS to rule attrs in a follow up CL.
 _COMMON_TEST_ATTRS = {
+    "collect_code_coverage": attr.bool(
+        doc = """
+Whether to collect code coverage for this test if `--collect_code_coverage=yes`.
```

Comment:
Want to mention that it's still only enabled with the normal Bazel flags as well?

Reply:
That's what I was going for with the mention of the flag. Do you have a suggestion on better wording?

Reply:
I swear I read these PRs. Misinterpreted the reference. 🤦‍♂️